### ISSUE TITLE
 "Default Noise Model in execute.py incorrectly uses depolarizing_error for amplitude_damping_error

### ISSUE THREAD

**Issue by GIRISHBELANI** [SRI-International/QC-App-Oriented-Benchmarks#475]
# Add amplitude damping error to all single qubit gates with error rate 0.0%
    #                         and to all two qubit gates with error rate 0.0%
    amp_damp_one_qb_error = 0.0
    amp_damp_two_qb_error = 0.0
    noise.add_all_qubit_quantum_error(depolarizing_error(amp_damp_one_qb_error, 1), ['rx', 'ry', 'rz'])    #line number = 193
    noise.add_all_qubit_quantum_error(depolarizing_error(amp_damp_two_qb_error, 2), ['cx'])                 #line number = 194
    -------------------------------------------------------------------------------------------------------------------------------------------------------
    # correction in above lines of code 

    # Add amplitude damping error to all single qubit gates with error rate 0.0%
    #                         and to all two qubit gates with error rate 0.0%
    amp_damp_one_qb_error = 0.0
    amp_damp_two_qb_error = 0.0
    noise.add_all_qubit_quantum_error(amplitude_damping_error(amp_damp_one_qb_error, 1), ['rx', 'ry', 'rz'])    #line number = 193
    noise.add_all_qubit_quantum_error(amplitude_damping_error(amp_damp_two_qb_error, 2), ['cx'])                 #line number = 194

**Comment by rtvuser1** [comment]
Yes, that does not look right.  Can you suggest a simple change that would fix this?

**Comment by GIRISHBELANI** [comment]
Instead of using depolarizing_error(), we can opt for **amplitude_damping_error()**. However, it only applies to **single-qubit errors**, as amplitude_damping_error() returns a quantum error channel specific to single-qubit generalized amplitude damping. Therefore, it **cannot be directly applied to a two-qubit error model** like amp_damp_two_qb_error.
